src/Molecules/Export/Export.jsx

```diff
@@ -127,11 +84,11 @@ const Export = (
       <Button
         color="success"
         data-testid="export-button"
-        onClick={() => onExport(filename, format, exportScope)}
+        onClick={onExport}
         disabled={
           filename === '' ||
-          (formats.length > 0 && format == null) ||
-          (extraOptions.length > 0 && exportScope == null)
+          (formats.length > 0 && (format == null || format === '')) ||

```

Comment:
Actually checks if format is in the formats list.
Incidentally also checks if the list is empty (`some` always returns false on empty arrays).
```suggestion
          !(formats || []).some((item = {}) => item.value === format) ||
```
Also for better readability, it can be extracted as a standalone function:
```js
const isAllowedFormat = (formats, format) => !(formats || []).some((item = {}) => item.value === format)
```